test/unit/test_run_reports.py

```diff
@@ -278,30 +270,7 @@ def test_run_reports_with_publish_failure(logging, emailer_send):
         call("Failed to publish report version r2-r2-version")
     ], any_order=False)
 
-    emailer_send.assert_has_calls([send_email_call_r1])
-
-
-@patch("src.utils.email.Emailer.send")
-def test_url_encodes_url_in_email(emailer_send):
```

Comment:
Moved to its own file (https://github.com/vimc/montagu-task-queue/pull/12/files#diff-bbeb6f05a2e0ce007a1a3dbe8c5a624eR10), now tests the `send_diagnostic_report_email` method as a unit rather than implicitly via the `run_reports` task